docs/core/setup/install-sdk.md

```diff
@@ -0,0 +1,144 @@
+
+You can also install .NET Core as part of an integrated development environment (IDE), detailed in the sections below.
+
+## Install with an installer
+
+Both Windows and macOS have standalone installers that can be used to install the .NET Core SDK.
+
+- Windows [x64 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-3.0.100-windows-x64-installer) | [x32 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-3.0.100-windows-x86-installer)
+- macOS [x64 CPUs](https://dotnet.microsoft.com/download/thank-you/dotnet-sdk-3.0.100-macos-x64-installer)
+
+::: zone pivot="os-linux"
+
+## Install with a package manager
+
+You can install .NET Core with many of the common linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-manager-rhel7.md).
+
+::: zone-end
+
+::: zone pivot="os-windows"
+
+## Visual Studio
+
+If you're using Visual Studio to develop .NET Core apps, the following table describes the minimum required version of Visual Studio based on the target .NET Core runtime.
+
+| .NET Core SDK version | Visual Studio version                      |
+| --------------------- | ------------------------------------------ |
+| 3.0                   | Visual Studio 2019 version 16.3 or higher. |
+| 2.2                   | Visual Studio 2017 version 15.9 or higher. |
+| 2.1                   | Visual Studio 2017 version 15.7 or higher. |
+
+If you already have Visual Studio installed, you can check your version with the following steps.
+
+01. Open Visual Studio.
+01. Select the **Help** menu > **About Microsoft Visual Studio**.
+01. Read the version number from the **About** dialog.
+
+### Download
+
+Visual Studio installs the latest .NET Core SDK and runtime, so you can target your app to run on the .NET Core runtime version of your choice.
+
+- [Download Visual Studio](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2019).
+
+Selecting any of the managed workloads for **Web**, **Desktop**, or **Azure development**, will install the .NET Core SDK development tools and runtime. Specifically choosing the **.NET Core** workload also installs .NET Core.
+
+[![Windows Visual Studio 2019 with .NET Core workload](media/install-sdk/windows-install-visual-studio-2019.png)](media/install-sdk/windows-install-visual-studio-2019.png#lightbox)
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+## Install with Visual Studio for Mac
+
+Visual Studio for Mac installs the .NET Core SDK when the **.NET Core workload** is selected. To get started with .NET Core development on macOS, see [Install Visual Studio 2019 for Mac](/visualstudio/mac/installation).
+
+[![macOS Visual Studio 2019 for Mac with .NET Core workload feature](media/install-sdk/mac-install-selection.png)](media/install-sdk/mac-install-selection.png#lightbox)
+
+::: zone-end
+
+## Install with Visual Studio Code
```

Comment:
I still feel this is a bit misleading. You're not installing with VS Code, you're using an installer. But I understand the intent behind this.